da_vinci-cdk/da_vinci_cdk/application.py

```diff
@@ -231,39 +260,57 @@ def __init__(self, app_name: str, deployment_id: str,
         else:
             s3_logging_bucket_name = None
 
+        resource_discovery_table_name = None
+
+        if resource_discovery_storage_solution not in [val for val in ResourceDiscoveryStorageSolution]:
```

Comment:
isn't ResourceDiscoveryStorageSolution an iterable? can't it be directly 
`if resource_discovery_storage_solution not in ResourceDiscoveryStorageSolution:` ?